pulsar-broker/src/test/java/org/apache/pulsar/broker/service/InactiveTopicDeleteTest.java

```diff
@@ -82,12 +84,14 @@ public void testDeleteWhenNoSubscriptions() throws Exception {
         consumer.close();
         producer.close();
 
-        Thread.sleep(2000);
+        Awaitility.await().atMost(2, TimeUnit.SECONDS).until(() -> admin.topics().getList("prop/ns-abc")
+                .contains(topic));
         Assert.assertTrue(admin.topics().getList("prop/ns-abc")
             .contains(topic));
```

Comment:
It can simplify by untiAssert(), Please check all.

Reply:
Thanks lhotari for the reminder